.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+
+jobs:
+  test:
+    docker:
+      - image: circleci/python:<< parameters.python >>
+    working_directory: ~/repo
```

Comment:
We tend to use the name of the project instead of `repo` (even if it shouldn't change anything)